docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -152,6 +152,13 @@ Search within 10,000 randomly generated options. The list is virtualized thanks
 
 {{"demo": "pages/components/autocomplete/Virtualize.js"}}
 
+## Limitations
+
+### iOS VoiceOver
+
+VoiceOver on iOS Safari doesn't support the `aria-owns` attribute very well.
```

Comment:
@fymmot do you know if there is a bug we can link, like https://bugs.webkit.org/show_bug.cgi?id=176896?

Reply:
@oliviertassinari I haven't been able to find a bug for this. I might file one myself.

There is some info in this article (https://tink.uk/using-the-aria-owns-attribute/), near the bottom concerning screen reader support. It's a little bit dated (2015) but there hasn't been any improvements for iOS VoiceOver since then.

Reply:
Missing data on a11ysupport.io: https://a11ysupport.io/tech/aria/aria-owns_attribute
powermapper.com: https://www.powermapper.com/tests/screen-readers/aria/

Concerning VoiceOver on iOS I wouldn't put too much time into it. According to powermapper their support is unreliable (59% vs >80% of jaws or nvda) and VoiceOver is stagnating at 12% (https://webaim.org/projects/screenreadersurvey8/) without data on iOS vs macOS for VoiceOver (so it's likely below 10% for VoiceOver + iOS)